src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on the "N+1" visit to a page, since previously
+// cached resources are updated in the background.
+
+// To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
```

Comment:
My first impression is that this might be overkill for this project. I'm not super familiar with doing this, so I'm not sure I want to maintain it. But perhaps the benefits outweigh the cost of maintaining. Is this rather boiler plate stuff that doesn't typically need adjusted or troubleshooting?